### DISCUSSION TITLE
Multicolumn Indices

### DISCUSSION THREAD

**Discussion by tobymao** [collerek/ormar#304] [Q&A] [answered]
Is there a way to define multi-column indices?

In SQLAlchemy it's normally done as

Index('myindex', mytable.c.col5, mytable.c.col6, unique=True)

**Comment by tobymao** [comment, marked as answer]
`  
 class Meta:

        constraints = [

            
              sqlalchemy.Index('idx_a_b', 'a', 'b')

        ]
`

seems to be what i want, is this correct?

**Reply by collerek** [reply]
Correct, that is what you are looking for but you need to use the ormar UniqueColumns and not the sqlalchemy index.
https://collerek.github.io/ormar/models/#constraints

**Reply by tobymao** [reply]
thanks

**Reply by tobymao** [reply]
but what if i don't want them to be unique?

**Reply by collerek** [reply]
Then it's an enhancement, should be relatively easy to add (in a similar fashion as UniqueColumn constraint).
You can open an enhancement issue if you need this one.
